src/app/components/calendar/calendarForm.tsx

```diff
@@ -48,10 +39,17 @@ const CalendarForm = () => {
     const fieldName = e.target.name;
     const fieldValue = e.target.value;
 
-    setCalendar((prevState) => ({
+    setCalendar((prevState: any) => ({
```

Comment:
Why any? :not-sure-if:

Reply:
Yeah, resolved in https://github.com/sarkaaa/paperprint-calendar/pull/6 .